### DISCUSSION TITLE
HALO CEVR OCULUS QUEST (RIGHTHAND) STEAM BINDINGS

### DISCUSSION THREAD

**Discussion by HoriZon-HZN** [LivingFray/HaloCEVR#30] [General]
Made my own custom Steam Bindings they work really good.

![halo-controls](https://github.com/user-attachments/assets/1aad005a-75d3-42ee-99eb-275c29d86dcd)

_____________________________________________________________________________
```
LivingFray VR mod v1.0.0 or higher

*Interact/Swap/Reload, RH Thumb Rest
*Jump/Brake (in a vehicle), A
*Throw Grenade, B
*Flashlight/Night Vision (scope), Y
*Switch Greande, X
*Crouch, RH Thumb Click (if set in VR cfg)
*Menu, LH Thumb Click

All other controls are mod defaults.
```
_____________________________________________________________________________

Also worth setting in the VR config;
**UIOverlayWidth = 800** (crosshair looks round instead of oval shaped)
**SnapTurn = false** (turns on smoothturn)
**SmoothTurnAmount = 160** (smoothturn speed I use)

**Comment by killacyst** [comment]
bro thanks for this

edit: 

*Crouch, RH Thumb Click (if set in VR cfg) <--- does that mean disable the crouch distance?

**Reply by HoriZon-HZN** [reply]
Yeah think its that one.
